src/include/ipc/header.h

```diff
@@ -154,6 +155,21 @@
 
 /** @} */
 
+/** \name DSP Command: Probes
+ *  @{
+ */
+
+#define SOF_IPC_PROBE_INIT			SOF_CMD_TYPE(0x001)
+#define SOF_IPC_PROBE_DEINIT			SOF_CMD_TYPE(0x002)
+#define SOF_IPC_PROBE_DMA_SET			SOF_CMD_TYPE(0x003)
+#define SOF_IPC_PROBE_DMA_GET			SOF_CMD_TYPE(0x004)
+#define SOF_IPC_PROBE_DMA_DETACH		SOF_CMD_TYPE(0x005)
+#define SOF_IPC_PROBE_POINT_SET			SOF_CMD_TYPE(0x006)
+#define SOF_IPC_PROBE_POINT_GET			SOF_CMD_TYPE(0x007)
+#define SOF_IPC_PROBE_POINT_REMOVE		SOF_CMD_TYPE(0x008)
```

Comment:
I've asked this question before....

Why can't the names for the IPC be symmetrical? the API should be self-explanatory and it's anything but that.

DMA_DETACH and POINT_REMOVE looks like oddities here...

There should be no need for explanations here. If it's not obvious what those IPCs do they definition is just wrong.

Please rework all this

Reply:
Unified to use 'remove' keyword

Reply:
So I'm seeing set, get and remove. I would expect to see "add" if we have a remove, but I'm guessing that "set" is adding a new probe ?

Reply:
Changed to `add`, `info` and `remove`

Reply:
your commit messsage still mentions GET, should be changed to INFO?

Reply:
fixd